src/main/java/frc/robot/Configs.java

```diff
@@ -5,52 +5,70 @@
+
+                        drivingConfig
+                                        .idleMode(IdleMode.kBrake)
+                                        .smartCurrentLimit(50);
+                        drivingConfig.encoder
+                                        .positionConversionFactor(drivingFactor) // meters
+                                        .velocityConversionFactor(drivingFactor / 60.0); // meters per second
+                        drivingConfig.closedLoop
+                                        .feedbackSensor(FeedbackSensor.kPrimaryEncoder)
+                                        // These are example gains you may need to them for your own robot!
+                                        .pid(0.04, 0, 0)
+                                        .velocityFF(drivingVelocityFeedForward)
+                                        .outputRange(-1, 1);
+
+                        turningConfig
+                                        .idleMode(IdleMode.kBrake)
+                                        .smartCurrentLimit(20);
+                        turningConfig.absoluteEncoder
+                                        // Invert the turning encoder, since the output shaft rotates in the opposite
+                                        // direction of the steering motor in the MAXSwerve Module.
+                                        .inverted(true)
+                                        .positionConversionFactor(turningFactor) // radians
+                                        .velocityConversionFactor(turningFactor / 60.0); // radians per second
+                        turningConfig.closedLoop
+                                        .feedbackSensor(FeedbackSensor.kAbsoluteEncoder)
+                                        // These are example gains you may need to them for your own robot!
+                                        .pid(1, 0, 0)
+                                        .outputRange(-1, 1)
+                                        // Enable PID wrap around for the turning motor. This will allow the PID
+                                        // controller to go through 0 to get to the setpoint i.e. going from 350 degrees
+                                        // to 10 degrees will go through 0 rather than the other direction which is a
+                                        // longer route.
+                                        .positionWrappingEnabled(true)
+                                        .positionWrappingInputRange(0, turningFactor);
+                }
+        }
+
+        public static final class ShooterConfigs {
+                public static final SparkMaxConfig shooterConfig = new SparkMaxConfig();
+
+                static {
+                        shooterConfig
+                                        .idleMode(IdleMode.kBrake)
+                                        .smartCurrentLimit(20);
```

Comment:
A NEO can take more current.
```suggestion
                                        .smartCurrentLimit(50);
```